standalone/src/models/cart/cart/shipping/shipping-draft/fields-config.ts

```diff
@@ -0,0 +1,30 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { AddressDraft, ReferenceDraft } from '@/models/commons';
+import type { TShippingDraftGraphql, TShippingDraftRest } from '../types';
+
+const commonFieldsConfig = {
+  key: fake((f) => f.lorem.slug(2)),
+  shippingAddress: fake(() => AddressDraft.random()),
+  shippingRateInput: null,
+  externalTaxRate: null,
+  deliveries: [],
+  custom: null,
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TShippingDraftRest> = {
+  fields: {
+    ...commonFieldsConfig,
+    shippingMethod: fake(() =>
+      ReferenceDraft.random().typeId('shipping-method')
+    ),
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TShippingDraftGraphql> = {
+  fields: {
+    ...commonFieldsConfig,
+    shippingMethod: fake(() =>
+      ReferenceDraft.random().typeId('shipping-method')
+    ),
+  },
```

Comment:
It seems we're using the same `shippingMethod` value in both configs. Wouldn't it make more sense to move it to the `commonFieldsConfig`?